getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "response = model.invoke(prompt_value)\n",
+    "response.content"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1ca904f4-faf1-4b95-bf77-7c7a7fa64515",
+   "metadata": {},
+   "source": [
+    "#### `ChatPromptTemplate`\n",
+    "\n",
+    "This is optimized for a conversation-like format. The prompt is a list of chat messages. Each chat message is associated with `role` and `content`. In the example code, `.from_messages` is used to include multiple messages.\n",
+    "\n",
+    "Here, we will hardcode roles in the chat prompt, as opposed to using `SystemMessage` or `HumanMessage` like earlier."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "46e869bf-370d-4742-837f-c7d5eb76a891",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_core.prompts import ChatPromptTemplate\n",
+    "\n",
+    "chat_template = ChatPromptTemplate.from_messages(\n",
+    "    [\n",
+    "        (\"system\", \"\"\"\n",
+    "                    You are a travel agent helping customers plan their trips.\n",
+    "                    Recommend them regarding popular Accommodation, Food and Activities of the country customer is asking.\n",
+    "                    \"\"\"), \n",
+    "        (\"ai\", \"Hi there, What can I help you with today?\"),\n",
+    "        (\"human\", \"Hi I'm {name}, I'm planning a trip to {country}. Any recommendations\")\n",
+    "    ]\n",
+    ")\n",
+    "\n",
+    "prompt_value = chat_template.format_messages(name=\"Lucy\", country=\"New Zealand\")\n",
+    "\n",
+    "# print(chat_template) # <- uncomment to see\n",
+    "# print(prompt_value) # <- uncomment to see"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "c87adc53-1248-441d-aa46-44f410c24796",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "\"Hi Lucy! New Zealand is an incredible destination with stunning landscapes, rich culture, and plenty of activities. Here are some recommendations for your trip:\\n\\n### Accommodation:\\n1. **Auckland:**\\n   - **SkyCity Hotel:** Located in the heart of the city, this hotel offers luxurious rooms and easy access to major attractions.\\n   - **Hotel DeBrett:** A boutique hotel with a unique design and a great location in the city center.\\n\\n2. **Queenstown:**\\n   - **Eichardt's Private Hotel:** Known for its elegance and stunning lake views.\\n   - **The Rees Hotel:** Offers luxurious accommodations with beautiful views of Lake Wakatipu.\\n\\n3. **Rotorua:**\\n   - **Regent of Rotorua:** A stylish boutique hotel with a central location.\\n   - **Solitaire Lodge:** A luxury lodge offering stunning views of Lake Tarawera.\\n\\n### Food:\\n1. **Auckland:**\\n   - **Depot Eatery & Oyster Bar:**\""
```

Comment:
```python
("system", """
                You're a travel agent helping customers plan their trips.
                Offer recommendations on natural features to visit, local cuisine, and activities based on the country the customer is asking about.
               """),
```

It will now only request natural locations to avoid calling out businesses or hotels :)